db/seeds.rb

```diff
@@ -16,6 +19,14 @@
   token.token = "attribute-service-token"
   token.save!
 
+  Doorkeeper::Application.create!(
+    name: "GOV.UK",
```

Comment:
This'll be the OAuth app used by account-api, so it'll do all GOV.UK personalisation.  The `redirect_uri` is going to finder-frontend currently because we've not yet done the card to move the callback page to frontend.

Reply:
Is the name already set, or can it be something more specific than "GOV.UK"?

Reply:
This is just the name in local dev, so it can be anything really.  Would "GOV.UK Personalisation" be better?

Reply:
Gotcha! And yes, "GOV.UK Personalisation" would be clearer I think 👍